### ISSUE TITLE
"TODO OVERDUE + DNP" Workflow pull queries with [[TODO]] as a parameter

### ISSUE THREAD

**Issue by Calhistorian** [roamhacker/SmartBlocks#32]
## 📋 Describe the new feature/idea/enhancement you are requesting

Exclude [[query]] from TODO pull workflows. When you pull TODOs that are OVERDUE + DNP, queries that had been used to keep track of TODOs are also pulled in the results of the workflow.

## 🔑 Why do you want this?

The workflow itself is for TODOs themselves, not queries which reference them.

## ⚠️ Is your feature request related to a problem? Please describe.

Don't want query blocks mixed in with TODO results.

## 💻 Environment

Platform: macOS 11.0.1

Browser: Chrome on 86.0.4240.198 (Official Build) (x86_64)

**Comment by roamhacker** [comment]
@Calhistorian I agree with you. I am seeing this in my test results and was already thinking we need to filter out some noise. Thank you for suggesting it. After the next release (in a few hours from this message), please try it again and see if you are satisfied with the results and then go ahead and close out the issue.

**Comment by roamhacker** [comment]
@Calhistorian just to mention, I am going to add this to all the <%TODO functions, I just don't see it makes sense.
